subprojects/testing-native/src/main/java/org/gradle/nativeplatform/test/xctest/SwiftCorelibXCTestSuite.java

```diff
@@ -14,13 +14,15 @@
  * limitations under the License.
  */
 
-package org.gradle.nativeplatform.fixtures.app
+package org.gradle.nativeplatform.test.xctest;
 
-import org.gradle.integtests.fixtures.SourceFile
+import org.gradle.api.Incubating;
 
-class SwiftXCTestBundleWithInfoPlist extends SwiftXCTestBundle {
-    @Override
-    List<SourceFile> getFiles() {
-        super.files + [emptyInfoPlist()]
-    }
+/**
+ * A XCTest suite targeting corelibs runtime of the API.
+ *
+ * @since 4.4
+ */
+@Incubating
+public interface SwiftCorelibXCTestSuite extends SwiftXCTestSuite {
```

Comment:
It's hard to say at this time. We could force it to one, but some developers may be depending on a custom `Info.plist` file. Would you suggest we just remove the capability of specifying a custom `Info.plist` and move forward with only a generated one? This would allow us to merge both types.